lib/jelix-modules/jacl2db/classes/jAcl2DbManager.class.php

```diff
@@ -514,4 +521,20 @@ public static function checkAclAdminRightsChanges(
 
         return self::ACL_ADMIN_RIGHTS_STILL_USED;
     }
+
+    public static function getHiddenRights() {
+        $config = jApp::config();
+        if (!property_exists($config, 'jacl2') || !array_key_exists('hiddenRights', $config->jacl2)
```

Comment:
to avoid to do these checks, put default values into lib/jelix/core/defaultconfig.ini.php.